devtools/gradle/src/main/java/io/quarkus/gradle/tasks/QuarkusDev.java

```diff
@@ -234,7 +234,7 @@ private QuarkusDevModeLauncher newLauncher() throws Exception {
                 .suspend(System.getProperty("suspend"));
         if (System.getProperty(IO_QUARKUS_DEVMODE_ARGS) == null) {
             builder.jvmArgs("-Dquarkus.test.basic-console=true")
-                    .jvmArgs("-Dio.quarkus.launched-from-ide=true");
+                    .jvmArgs("-Dio.quarkus.io.quarkus.force-color-support=true");
```

Comment:
Is the duplicate `io.quarkus.io.quarkus` intentional here? :-)

Reply:
no